data/json/recipes/armor/arms.json

```
@@ -38,6 +38,10 @@
     "time": "390 m",
     "autolearn": true,
     "byproducts": [ [ "scrap_leather", 80 ] ],
+    "proficiencies": [
+      { "proficiency": "prof_chitinworking", "required": false, "time_multiplier": 3 },
+      { "proficiency": "prof_leatherworking", "required": false, "time_multiplier": 3 }
```

```suggestion
      { "proficiency": "prof_leatherworking", "required": false, "time_multiplier": 2 },
      { "proficiency": "prof_articulation", "required": false, "time_multiplier": 2 }
```
I'd reduce the impact of leather in this case because it's probably just straps and things, and I'd add articulation because this one is part of a larger suit of armour and so probably goes over the elbow, and maybe shoulder and wrist as well.